templates/OperationUrlSnippet.stencil

```diff
@@ -5,7 +5,11 @@ guard let urlTemplate = "{{ op.request.path }}".removingPercentEncoding else {
 }
 let pathParams = [
     {% for param in op.params.path %}
+    {% if param.implementation == "client" %}
+        "{{ param.key }}" : client.{{ param.value}},
+    {% else %}
         "{{ param.key }}" : {{ param.value}},
+    {% endif %}
```

Comment:
Same here, we can eliminate the if/else logic if we build up the value as a path.

Reply:
Removed in new iteration.